lib/constants.ts

```diff
@@ -1,3 +1,29 @@
+import { StyleSheet } from "react-native";
+
+// TODO: check dark mode shadows in ios
+export const SHADOWS = StyleSheet.create({
+  medium: {
+    elevation: 2,
+    shadowColor: "black",
+    shadowOpacity: 0.15,
+    shadowOffset: {
+      width: 2,
```

Comment:
minor: 0 for the width offset looks nicer IMO so it's just a downward shadow. Maybe it is just me though?